src/app/plugins/register.plugin.ts

```diff
@@ -17,16 +17,40 @@ export class RegisterPlugin extends Plugin {
   }
 
   public async execute(message: IMessage, args?: string[]) {
-    const request = this.container.classService.buildRequest(message.author, args);
-    if (!request) {
-      message.reply('I was unable to build your request.');
+    if (!args) {
       return;
     }
-    try {
-      const response = await this.container.classService.register(request);
-      message.reply(response);
-    } catch (e) {
-      message.reply(e);
+
+    const successfulClasses: string[] = [];
+    const invalidClasses: string[] = [];
+    for (const arg of args) {
+      const request = this.container.classService.buildRequest(message.author, [arg]);
+      if (!request) {
+        invalidClasses.push(arg);
+        continue;
+      }
+      try {
+        const response = await this.container.classService.register(request);
+        if (response.includes('success')) {
+          successfulClasses.push(arg);
+        } else {
+          invalidClasses.push(arg);
+        }
+      } catch (e) {
+        message.reply(e);
+      }
```

Comment:
we dont want this to happen n times.

Reply:
Should I just push the arg to `invalidClasses` and `this.container.loggerService.error(e)`?